internal/server/server.go

```diff
@@ -48,12 +48,17 @@ func New(logger log.Logger, reg *prometheus.Registry, opts ...Option) Server {
 		promhttp.InstrumentMetricHandler(reg, promhttp.HandlerFor(reg, promhttp.HandlerOpts{})).ServeHTTP(w, r)
 	})
 
-	r.Get("/prometheus/query",
-		ins.newHandler("query",
-			proxy.New(logger, "/prometheus/query", options.metricsQueryEndpoint, options.proxyOptions...)))
-	r.Post("/prometheus/write",
-		ins.newHandler("write",
-			proxy.New(logger, "/prometheus/write", options.metricsWriteEndpoint, options.proxyOptions...)))
+	queryEndpoint := "/api/v1/metrics/query"
+	r.Get(queryEndpoint,
+		ins.newHandler("query", proxy.New(logger, queryEndpoint, options.metricsQueryEndpoint, options.proxyOptions...)))
+
+	queryRangeEndpoint := "/api/v1/metrics/query_range"
+	r.Get(queryEndpoint,
+		ins.newHandler("query_range", proxy.New(logger, queryRangeEndpoint, options.metricsQueryEndpoint, options.proxyOptions...)))
+
+	receivePath := "/api/v1/metrics/receive"
```

Comment:
I realize that the receive component used `receive` here, but I think I actually now like `write` better. Thoughts?

Reply:
sgtm. I'll revert it. Somehow receive sounds better for me an instant moment :D